compiler/AST/checkAST.cpp

```diff
@@ -214,8 +216,6 @@ void checkPrimitives()
      case PRIM_ARRAY_SHIFT_BASE_POINTER:
      case PRIM_ARRAY_SET:
      case PRIM_ARRAY_SET_FIRST:
-     case PRIM_ERROR:
-     case PRIM_WARNING:
```

Comment:
Do you know what about your PR caused prim_error and prim_warning to still exist during resolution? Nothing jumps out at me.